slices/vim-tiny.yaml

```diff
@@ -0,0 +1,25 @@
+package: vim-tiny
+
+essential:
+  - vim-tiny_copyright
+
+slices:
+  bins:
+    essential:
+      - libacl1_libs
+      - libc6_libs
+      - libselinux1_libs
+      - libtinfo6_libs
+      - vim-common_config
+      - vim-tiny_config
+    contents:
+      /usr/bin/vi: {symlink: /usr/bin/vim.tiny}
```

Comment:
Coming back to this again, `vim` also tries to create a symlink from `vi`:

```sh
update-alternatives --install /usr/bin/vi vi /usr/bin/vim.basic 30
```

Perhaps we should ignore making the `vi` symlink?